#!/bin/sh /usr/share/dpatch/dpatch-run
## 01-manpage.dpatch by Daniel Baumann <daniel@debian.org>
##
## DP: Using (C) instead of the copyright-symbol.

@DPATCH@

diff -Naur micro_httpd.orig/micro_httpd.8 micro_httpd/micro_httpd.8
--- micro_httpd.orig/micro_httpd.8	2005-05-20 19:10:23.000000000 +0000
+++ micro_httpd/micro_httpd.8	2005-11-05 13:29:48.769796752 +0000
@@ -66,7 +66,7 @@
 Make sure the paths to the certificate and executable are correct, and
 again don't forget to change "dir" to the directory you want to serve.
 .SH AUTHOR
-Copyright © 1999 by Jef Poskanzer <jef@mail.acme.com>. All rights reserved.
+Copyright (C) 1999 by Jef Poskanzer <jef@mail.acme.com>. All rights reserved.
 .\" Redistribution and use in source and binary forms, with or without
 .\" modification, are permitted provided that the following conditions
 .\" are met:
